script1.sh

```diff
@@ -0,0 +1,11 @@
+echo "Dvir Avraham" > ~/output.txt
+printf "DvirAvraham" > ~/filename.txt
+mkdir ~/new
+cp ~/filename.txt ~/new/
+mv ~/new/filename.txt ~/new/Dvir_Avraham.txt
+gedit ~/new/your_name.txt
+cp ~/new/Dvir_Avraham.txt ~/new/newFile
+ls ~/new
+cd
+cd ~
+rm -r ~/new
```

Comment:
fix the command to be rm -rf new will recursively delete the directory new and all of its contents.
 The -r flag tells rm to recursively delete directories, and the -f flag tells rm to force the deletion, even if the directory is not empty.